resources/wikia/libraries/jquery/autocomplete/jquery.autocomplete.js

```diff
@@ -293,19 +305,28 @@
         return;
       }
 
-      var me, len, div, f, suggestion;
+      var me, len, div, f, suggestion, v, suggestionWrapperElement;
       me = this;
       len = this.options.maxSuggestions && this.options.maxSuggestions < this.suggestions.length ? this.options.maxSuggestions : this.suggestions.length;
       f = this.options.fnFormatResult;
       v = this.getQuery(this.currentValue);
+      // wikia change - start
+      suggestionWrapperElement = this.options.suggestionWrapperElement;
+      // wikia change - end
       this.container.hide().empty();
 
       for (var i = 0; i < len; i++) {
-          // wikia change - start
-          suggestion = this.suggestions[i];
-        div = $((me.selectedIndex === i ? '<div class="' + this.options.selectedClass + '"' : '<div')
-            + ' title="' + $.htmlentities(suggestion) + '">' + f($.htmlentities(suggestion), this.data[i], $.htmlentities(v))
-            + '</div>');
+        // wikia change - start
+        suggestion = this.suggestions[i];
+        div = $(
+          (
+            me.selectedIndex === i ?
+            '<' + suggestionWrapperElement + ' class="' + this.options.selectedClass + '"' :
+            '<' + suggestionWrapperElement
+          )
+          + ' title="' + $.htmlentities(suggestion) + '">' + f($.htmlentities(suggestion), this.data[i], $.htmlentities(v))
+          + '</' + suggestionWrapperElement + '>'
+        );
```

Comment:
How do you like it now? Is it correct? https://github.com/Wikia/app/pull/11295/commits/703450708715529aefffff5b4c10eb2edc167a6c